nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>That's not quite true, and isn't really documenting this package. Instead, explain something like what I mentioned in the previous comment. Note that you'll need to move your example down to beneath the <code>def dcmread</code>  otherwise it won't work!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Moved the cells so now there is no issue with calling <code>dcmread</code>  and removed this comment</p>